docs/source/configuration.rst

```diff
@@ -57,6 +57,76 @@ Cloud provider configuration
     [cloud-provider]
     gcp-zone = us-east4-b
 
+.. _elb_aws_region:
+
+``AWS Region``
+^^^^^^^^^^^^^^
+
+    Name of the AWS region to use. Recommended value: ``us-east-1``.
+
+    * Default: None
+    * Values: String, any region that supports Batch, see `AWS documentation for details <https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/>`_
+
+    Also supported via the environment variable: ``ELB_GCP_REGION``.
+
+    For background information about AWS regions, please see the `AWS
+    documentation
+    <https://aws.amazon.com/about-aws/global-infrastructure/regions_az/>`_.
+
+.. code-block::
+
+    [cloud-provider]
+    aws-region = us-east-1
+
+.. _elb_aws_subnet:
+
+``AWS Subnet``
+^^^^^^^^^^^^^^
+
+    Name of the AWS subnet to use; must exist in the chosen :ref:`region
+    <elb_aws_region>`.
+
+    * Default: None
+    * Values: String
+
+.. code-block::
+
+    [cloud-provider]
+    aws-subnet = subnet-a43744d3
```

Comment:
I am wondering if we are revealing any secrets about NCBI AWS account here.

Reply:
These are randomly generated :)